core/box.h

```diff
@@ -502,6 +502,18 @@ typedef struct {
     uint32_t reserved2[4];
 } isom_dovi_t;
 
+typedef struct {
+    ISOM_BASEBOX_COMMON;
+    uint8_t configurationVersion;
+    uint16_t min_spatial_segmentation_idc;
```

Comment:
This is the only snake case field in the struct, all others are camel case. Is this intentional? It looks like they should all be snake case to be consistent with the surrounding box structs.

Reply:
These names are directly from the spec... you can blame the authors.

Reply:
I see other boxes have similar inconsistencies so I guess it's too late to bikeshed this.